packages/venia-ui/lib/components/SearchPage/searchPage.js

```diff
@@ -8,25 +8,19 @@ import Gallery from '../Gallery';
 import FilterModal from '../FilterModal';
 import { fullPageLoadingIndicator } from '../LoadingIndicator';
 import Pagination from '../../components/Pagination';
-import CategoryFilters from './categoryFilters';
 import defaultClasses from './searchPage.css';
 import PRODUCT_SEARCH from '../../queries/productSearch.graphql';
+import FilterIntrospectionQuery from '../../queries/introspection/filterIntrospectionQuery.graphql';
 
 const SearchPage = props => {
     const classes = mergeClasses(defaultClasses, props.classes);
 
     const talonProps = useSearchPage({
+        FilterIntrospectionQuery,
```

Comment:
Sorry to nitpick, but the naming of these things is inconsistent.

- By convention, props are camelCase
- In this same file, imported queries are ALL_CAPS

```es6
// option A:
// fully name query props when there's more than one
useSearchPage({
  filterIntrospectionQuery: FILTER_INTROSPECTION,
  productSearchQuery: PRODUCT_SEARCH
})

// option B:
// define `queries` outside the component, or memoize with `useMemo`
const queries = {
  FILTER_INTROSPECTION,
  PRODUCT_SEARCH
}

useSearchPage({ queries })
```

My preference here would be option B.

Reply:
Oops you are right - I got myself into the practice of using a different style in the cart page. I'll fix these :)